src/DatePicker/DatePicker.js

```diff
@@ -350,12 +357,18 @@ DatePicker.propTypes = {
     buttonProps: PropTypes.object,
     compact: PropTypes.bool,
     enableRangeSelection: PropTypes.bool,
-    inputProps: PropTypes.object
+    inputProps: PropTypes.object,
+    onBlur: PropTypes.func
+};
+
+DatePicker.defaultProps = {
+    onBlur: () => {}
 };
 
 DatePicker.propDescriptions = {
     ...Calendar.propDescriptions,
-    enableRangeSelection: 'Set to **true** to enable the selection of a date range (begin and end).'
+    enableRangeSelection: 'Set to **true** to enable the selection of a date range (begin and end).',
+    onBlur: 'Callback function for onBlur events. In the object returned, date is the date object and formattedDate is the formatted date.'
```

Comment:
Maybe add backticks around `date` and `formattedDate` when describing the returned object.